src/main/java/mops/presentation/DirectoryController.java

```diff
@@ -59,21 +62,21 @@ public String showFolderContent(KeycloakAuthenticationToken token,
     /**
      * Uploads a file.
      *
-     * @param token    keycloak auth token
-     * @param model    spring view model
-     * @param dirId    id of the directory id where it will be uploaded
-     * @param fileInfo file object
+     * @param token         keycloak auth token
+     * @param model         spring view model
+     * @param dirId         id of the directory id where it will be uploaded
+     * @param multipartFile file object
      * @return route after completion
      */
     @PostMapping("/{dirId}/upload")
-    @SuppressWarnings({ "PMD.DataflowAnomalyAnalysis", "PMD.EmptyCatchBlock" })
+    @SuppressWarnings({"PMD.DataflowAnomalyAnalysis", "PMD.EmptyCatchBlock"})
```

Comment:
Wrong Formatting - 
```suggestion
    @SuppressWarnings({ "PMD.DataflowAnomalyAnalysis", "PMD.EmptyCatchBlock" })
```
would be correct here - problems exists multiple times

Reply:
yes I didn't get the info that you have to change it in intellij